tests/test_blueprints.py

```diff
@@ -899,3 +899,51 @@ def grandchild_no():
     assert client.get("/parent/no").data == b"Parent no"
     assert client.get("/parent/child/no").data == b"Parent no"
     assert client.get("/parent/child/grandchild/no").data == b"Grandchild no"
+
+
+def test_nested_blueprint_url_prefix(app, client):
```

Comment:
This test seems too convoluted for what it tests. Could you simplify it to test the prefix alone - for example using the snippet in #4037?

Reply:
Sure. Just updated.